framework/auth/__init__.py

```diff
@@ -48,6 +49,7 @@ def logout():
             del session.data[key]
         except KeyError:
             pass
+    Session.remove(Q('_id', 'eq', session._id))
```

Comment:
Probably the latter, but I'm not sure I understand the consequences of the difference.